pages/landing/migrate-from-heroku/pricing/index.html

```diff
@@ -0,0 +1,85 @@
+---
+layout: landing
+type: landing
+slug: index-page
+
+title: Pricing
+excerpt: Configure the plan just right for your team.
+permalink: /migrate-from-heroku/pricing/
+
+pricing:
+  - title: Free
+    description: For solopreneuers and personal projects.
+    price: Free
+    image: /assets/img/grunty-free@2x.png
+    image_styles: "max-width: 150px; margin-top: 18px"
+    features:
+      - name: 1 App
+        description: An app is a single service that you want to migrate from Heroku to AWS. This plan allows you to migrate & run only one app. For more apps, please see the other plans.
+      - name: Single AWS Account
+        description: You can only migrate to a single AWS account with this plan. For multi-account setup, please refer to the Enterprise plan.
+      - name: 1 User
+        description: A user is a human or machine that accesses the Gruntwork code repos, training materials, or support services. For more than 10 users, see Gruntwork Enterprise.
+      - name: Gruntwork Lite Stack
+        description: You get access to the Gruntwork Lite Stack, which includes modules for services such as ECS Fargate, ALB, ECR and Aurora Serverless.
+      - name: Access to Docs
+        description: This plan gets you access to all the modules documentation and any production guides for free. For access to the DevOps Training Library, see the Enterprise plan.
+      - name: $0 USD estimated AWS costs*
+        description: This plan is eligible for the AWS Free Tier. Please contact us for more information.
+
+  - title: Pro
+    description: For startups, small businesses planning on running a few apps.
+    price: "495"
+    image: /assets/img/grunty-pro@2x.png
+    image_styles: "max-width: 150px; margin-top: 18px"
+    features:
+      - name: Up to 10 Apps
+        description: An app is a single service that you want to migrate from Heroku to AWS. This plan allows you to migrate & run up to 10 apps. For more, please see the Enterprise plan.
+      - name: Single AWS Account
+        description: You can only migrate to a single AWS account with this plan. For multi-account setup, please refer to the Enterprise plan.
+      - name: Up to 10 Users
+        description: A user is a human or machine that accesses the Gruntwork code repos, training materials, or support services. For more than 10 users, see Gruntwork Enterprise.
+      - name: Gruntwork Pro Stack
+        description: You get access to the Gruntwork Pro Stack, which includes modules from the Gruntwork Lite Stack, and RDS, ElastiCache, Elasticsearch.
+      - name: Access to Docs
```

Comment:
I think training videos could be in the free plan too.

Reply:
@brikis98 are you referring to the DevOps Training Library? I think this starts to get a bit fuzzy as we dilute the value prop of our existing customers paying us $795/mo.

Reply:
+1 on both points here actually. Following our conversation on Friday, @robmorgan, we agreed that it needs to be cheaper than the current minimum price. But I then had a realisation yesterday evening - they're not actually paying for the same product at all. Here, I assume it's a one time feature use, where they get to keep the code we generate for them - but we don't grant them access to all modules, or the core ones. Does that take us in the right direction?

Reply:
Yes, the DevOps Training Library.

What I'm struggling with is that our product is already hard to use, so if we deny a large subset of our customers access to training/learning materials, that only gets worse.

Reply:
what's our decision here? I think we can safely make it higher, but still agree with Rob that we shouldn't make it as high as the monthly subscription for IaC now